src/ray/gcs/gcs_client/gcs_client.h

```diff
@@ -81,9 +85,11 @@ class RAY_EXPORT GcsClient : public std::enable_shared_from_this<GcsClient> {
   /// Connect to GCS Service. Non-thread safe.
   /// This function must be called before calling other functions.
   /// \param instrumented_io_context IO execution service.
+  /// \param cluster_id Optional cluster ID to provide to the client.
```

Comment:
should we add comments somewhere on expected behavior of setting cluster_id to Nil?